### ISSUE TITLE
Wrong package.json/tsconfig.json configuration

### ISSUE THREAD

**Issue by cxrtisxl** [k0rean-rand0m/rarible-serverside#1]
Got an issue trying to run the project, because it compiles with "require"s which are not supported by ES modules. `package.json` and `tsconfig.json` should be properly configured.

```
> node dist/main.js

/Users/ian/git/rarible-serverside/dist/main.js:25
const node_fetch_1 = __importDefault(require("node-fetch"));
                                     ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/ian/git/rarible-serverside/node_modules/node-fetch/src/index.js from /Users/ian/git/rarible-serverside/dist/main.js not supported.
Instead change the require of index.js in /Users/ian/git/rarible-serverside/dist/main.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/Users/ian/git/rarible-serverside/dist/main.js:25:38) {
  code: 'ERR_REQUIRE_ESM'
}

Node.js v17.7.2
```

**Comment by cxrtisxl** [comment]
The possible solution is to use node-fetch V2 (not V3). Thanks to [@dobrokhvalov](https://github.com/dobrokhvalov) for figuring it out!

**Comment by cxrtisxl** [comment]
The issue is solved with using node-fetch V2 🔥
Commit with the fix: [07400a9](https://github.com/k0rean-rand0m/rarible-serverside/commit/07400a9a9570a9f21847e6f86bea3d3b5be5f179)
